styles/components/factsheets/area-table.scss

```diff
@@ -100,4 +92,8 @@
       }
     }
   }
+
+  &.-actors {
+    margin-top: 56px;
```

Comment:
Don't set position, margin, size properties here. Just common properties. Other ones go to layouts.